testsuite/shared/src/main/java/org/jboss/as/test/shared/LayersTestBase.java

```diff
@@ -249,7 +250,7 @@ public abstract class LayersTestBase {
             "ibm.jdk",
             "javax.api",
             "javax.sql.api",
-            "javax.xml.stream.api",
+            "javax.xml.stream.api"
```

Comment:
@fjuma Thank you for submitting this PR fix! I noticed there is a comma missing so the build will fail

Reply:
@Skyllarr Thanks very much for spotting that! I've fixed this.